src/utils/download.ts

```diff
@@ -1,24 +1,46 @@
 import * as fs from 'fs';
 import path from 'path';
-import axios, { AxiosResponse } from 'axios';
+import axios from 'axios';
+import { uniqueId } from 'lodash';
```

Comment:
I think we can just do Number.random() or something native. We currently only include `lodash` because our seed phrase package would throw compilation errors